templates/report.html.jinja

```diff
@@ -116,13 +116,50 @@ Monthly GTFS Quality Reports: {{parameters["DATE_MONTH_YEAR"]}} {{parameters["AG
                 {% endfor %}
             </tbody>
 
+            <!-- validation severity errors -->
+
+            {% if validation_severity is not defined %}
+
```

Comment:
when does this case happen and is totally blank the right answer? shouldn't this case lead to the "Not available" message under the else?

Reply:
This is correct per a conversation between me, Andrew, and Natalya last week -- this is a new table that uses new data that wasn't defined for prior months. The condition is constructed such that all reports prior to March 2022 do not have their appearance altered (they don't have a message added to preserve their original format)